trogdor/src/test/java/org/apache/kafka/trogdor/common/JsonSerializationTest.java

```diff
@@ -74,7 +74,7 @@ private <T> void verify(T val1) throws Exception {
         Class<T> clazz = (Class<T>) val1.getClass();
         T val2 = JsonUtil.JSON_SERDE.readValue(bytes, clazz);
         for (Field field : clazz.getDeclaredFields()) {
-            boolean wasAccessible = field.isAccessible();
+            boolean wasAccessible = field.canAccess(val2);
```

Comment:
`.isAccessible()` has been deprecated since Java 9.
The documentation recommends using `.canAccess(object)` instead.